chameleon/settings.py

```diff
@@ -150,13 +152,28 @@
     # djangocms-blog
     "filer",
     "easy_thumbnails",
+    "mptt",
```

Comment:
what is this for?

Reply:
It's a dependency of Filer, which is a dependency of bootstrap 4: https://django-filer.readthedocs.io/en/latest/installation.html